docs/ref-manual/cli/options.md

```diff
@@ -399,6 +399,53 @@ Where `TokenPair` is
 We have two contracts `BankToken.sol` and `LoanToken.sol`. We want `tokenA` of the `tokenPair` to be `BankToken`, and `tokenB` to be `LoanToken`. Addresses take up only one slot. We assume `tokenPair` is the first field of Bank (so it starts at slot zero). To do that, we use:  
 `certoraRun Bank.sol BankToken.sol LoanToken.sol --verify Bank:Bank.spec --structLink Bank:0=BankToken Bank:1=LoanToken`
 
+Options for controlling contract creation
+-----------------------------------------
+
+### `--dynamic_bound`
+
+**What does it do?**
+If set to zero (the default) dynamic creation will not be modeled. If non-zero, then dynamic contract creation will be modeled with cloning, where each contract will be cloned at most n times.
+**When to use it?**
+When you wish to model contract creation as a non-havoc.
+**Example**
+Suppose a contract `C` creates a new instance of a contract `Foo`, and you wish to inline the constructor of `Foo` at the creation site.
+`certoraRun C.sol Foo.sol --dynamic_bound 1`
+
+### `--dynamic_dispatch`
+
+**What does it do?**
+If false (the default), then all contract method invocations on newly created instances on spec must be modeled with explicit DISPATCHER summaries. 
```

Comment:
I think the first mention of `DISPATCHER` summaries should be a link to the corresponding page.  I think you can write ``` {ref}`` `DISPATCHER` <dispatcher>`` ``` To make a reference with code font text.

Reply:
It's worth noting the limitations of this flag: it *only* does something when the prover can *prove* that the callee is a dynamically created contract. That is, if we have:

```
MyFoo f;
if(*) {
   f = new MyFoo(...);
} else {
  f = storageStruct.myFoo;
}
f.bar();
```

the automatic dispatcher won't do anything